react/react1/week2/Todolist/src/components/todoApp/App.jsx

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'react';
+
+import TodoAppList from '../todoAppList/todoAppList'
+import TodoForm from '../todoForm/todoForm';
+import TodoTimer from '../todoTimer/todoTimer';
+import './App.css'
+
+import todos from '../../todos';
+
+
+class App extends Component {
```

Comment:
This should be rewritten as a function component. Thy rewriting this as a function with useState states and a return